app/emoji.go

```diff
@@ -37,7 +38,7 @@ const (
 	MaxEmojiOriginalHeight = 1028
 )
 
-func (a *App) CreateEmoji(sessionUserId string, emoji *model.Emoji, multiPartImageData *multipart.Form) (*model.Emoji, *model.AppError) {
+func (a *App) CreateEmoji(c request.CTX, sessionUserId string, emoji *model.Emoji, multiPartImageData *multipart.Form) (*model.Emoji, *model.AppError) {
```

Comment:
Could we drop the `sessionUserId`, it's already conveyed in the context right?

Reply:
Yes. In a later step I want to provide additional methods for that. Most (all?) calls that check for permissions are looking for the current session. Providing the context *and* session ID is redundant.
I wanted to focus purely on adding the context for this (and the following) PR(s).